client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -1943,6 +1944,22 @@ public void testVerifyRepository() {
         assertThat(expectedParams, equalTo(request.getParameters()));
     }
 
+    public void testCreateSnapshot() throws IOException {
+        Map<String, String> expectedParams = new HashMap<>();
+        String repository = randomIndicesNames(1, 1)[0];
+        String snapshot = "snapshot-" + generateRandomStringArray(1, randomInt(10), false, false)[0];
+        String endpoint = "/_snapshot/" + repository + "/" + snapshot;
+
+        CreateSnapshotRequest createSnapshotRequest = new CreateSnapshotRequest(repository, snapshot);
+        setRandomMasterTimeout(createSnapshotRequest, expectedParams);
```

Comment:
can u also add a set to the withWaitForCompletion

Reply:
Good call, added.